docs/source/fitting/fitfunctions/AsymmetricPearsonVII.rst

```diff
@@ -0,0 +1,81 @@
+.. _func-AsymmetricPearsonVII:
+
+======================
+Asymmetric Pearson VII
+======================
+
+.. index:: AsymmetricPearsonVII
+
+Description
+-----------
+
+The asymmetric Pearson VII function (sometimes it is also referred to as the split-Pearson VII function) is a function that combines
+two Pearson VII distributions so to fit sharp peak curves. It is useful for analysis of X-ray diffraction peaks which consist of two separable components.
+
+In one of the representations, the asymmetric Pearson VII function :math:`A(x)` can be written as a superposition of the "left" and "right" Pearson VII functions :math:`P_L(x)` and :math:`P_R(x)`,
+
+.. math:: A(x) = P_L(x) \, \theta(x_0 - x) + P_R(x) \, \theta(x - x_0),
+
+where :math:`x_0` denotes the coordinate of the peak centre, :math:`\theta` denotes the Heavyside step function with :math:`\theta(y \ge 0) = 1` and :math:`\theta(y < 0)=0`, and :math:`P(x)` denotes the Pearson VII function. This function takes four additional parameters and in general is defined as
```

Comment:
In the above equation there is noting that would suggest X_0 has to be a peak centre. I think it would be better to say something like:

where, :math:'x_0' is the the value when the function changes between the left and right Pearson VII functions, this is usually chosen to be the peak centre.